cmd/kritis/gcbsigner/main.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/golang/glog"
+	"github.com/grafeas/kritis/pkg/kritis/crd/buildpolicy"
+	"github.com/grafeas/kritis/pkg/kritis/gcbsigner"
+	"github.com/grafeas/kritis/pkg/kritis/metadata/containeranalysis"
+	"github.com/grafeas/kritis/pkg/kritis/secrets"
+	"golang.org/x/net/context"
+
+	"cloud.google.com/go/pubsub"
+)
+
+func main() {
+	gcbProject := flag.String("gcb_project", "", "Id of the project running GCB")
```

Comment:
Nothing to fix here, but this is the first time I've seen flags defined in main() rather than init(). It confused me on first read, but looks like a decent pattern for keeping the scope small.